tests/nodeToJsxNode.test.js

```
@@ -0,0 +1,28 @@
+import nodeToJsxNode from '../src/nodeToJsxNode';
+
+describe('htmlToReactAttributes', () => {
+  it('attributes should change to jsx', () => {
+    const input = '<path fill-rule="evenodd" />';
+    document.body.innerHTML = input;
+    expect(nodeToJsxNode(document.body.children[0])).toBe('<path fillRule="evenodd" ></path>');
+    expect(nodeToJsxNode(document.body.children[0])).toMatchSnapshot();
+  });
+  it('attributes should stay the same', () => {
+    const input = '<path fill="#232" />';
+    document.body.innerHTML = input;
+    expect(nodeToJsxNode(document.body.children[0])).toBe('<path fill="#232" ></path>');
+    expect(nodeToJsxNode(document.body.children[0])).toMatchSnapshot();
+  });
+  it('attribute class should change to className', () => {
+    const input = '<path class="math" />';
+    document.body.innerHTML = input;
+    expect(nodeToJsxNode(document.body.children[0])).toBe('<path className="math" ></path>');
+    expect(nodeToJsxNode(document.body.children[0])).toMatchSnapshot();
+  });
+  it('it should only return a single element as string', () => {
```

If the desired behavior is to only transform and return the first tag I think we should throw an error if more than 1 element is passed in. Then test that case, pass in 2 elements and catch the error.  This would make the function more predictable.

Given `<path class="math"/><path class="math"/>` as input, I find it a bit confusing that 1 element is returned.

In case future changes were made with the assumption that the function would transform multiple tags, we could delete this behavior and add tests around the new functionality.